coordinator/src/routes/prescribe/prepare.ts

```diff
@@ -11,7 +14,7 @@ export default [
   {
     method: "POST",
     path: `${BASE_PATH}/$prepare`,
-    handler: userAuthValidator(externalValidator(
+    handler: externalValidator(
```

Comment:
Is `userAuthValidator` used anywhere now or can it just go?

Reply:
ill delete :+1: